cmd/api/src/database/sso_providers.go

```diff
@@ -99,6 +100,13 @@ func (s *BloodhoundDB) GetAllSSOProviders(ctx context.Context, order string, sql
 		query = query.Where(sqlFilter.SQLString, sqlFilter.Params...)
 	}
 
+	// Backwards compatibility when FF is disabled
+	if oidcFeatureFlag, err := s.GetFlagByKey(ctx, appcfg.FeatureOIDCSupport); err != nil {
```

Comment:
Probably fine since this DB call is only used for our feature, but I'm thinking we'd probably want this in the endpoint rather than in the database call.

In my eyes the database call is doing what you're asking it, just grabbing SSOProviders, but it's the actual endpoint that implements the feature that is meant to be feature gated.

Reply:
We started there but shoehorning in the FF check where it properly bubbled down to this layer got complicated with how the handlers parse filtering whereas this layer offered a quick and dirty workaround in the interim